lib/pages/carteiraApp.dart

```diff
@@ -0,0 +1,177 @@
+                            Text(
+                              '+' + tabela[0].variacao.toString() + '%',
+                              style: TextStyle(
+                                  fontSize: 10,
+                                  fontWeight: FontWeight.bold,
+                                  color: (tabela[0].variacao > 0)
+                                      ? Colors.green
+                                      : Colors.red,
+                                  backgroundColor: (tabela[0].variacao > 0)
+                                      ? Colors.green[50]
+                                      : Colors.red[50]),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  ),
+                  Container(
+                    child: ListTile(
+                      leading: Image.asset(
+                        tabela[1].icone,
+                        width: 30,
+                      ),
+                      title: Text(tabela[1].sigla),
+                      subtitle: Text(tabela[1].moeda),
+                      trailing: AnimatedOpacity(
+                        opacity: opacidade ? 1 : 0,
+                        duration: Duration(milliseconds: 800),
+                        child: Column(
+                          children: [
+                            Text(tabela[1].valor),
+                            Text(
+                              '+' + tabela[1].variacao.toString() + '%',
+                              style: TextStyle(
+                                  fontSize: 10,
+                                  fontWeight: FontWeight.bold,
+                                  color: (tabela[1].variacao > 0)
+                                      ? Colors.green
+                                      : Colors.red,
+                                  backgroundColor: (tabela[1].variacao > 0)
+                                      ? Colors.green[50]
+                                      : Colors.red[50]),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  ),
+                  Container(
+                    child: ListTile(
+                      leading: Image.asset(
+                        tabela[2].icone,
+                        width: 30,
+                      ),
+                      title: Text(tabela[2].sigla),
+                      subtitle: Text(tabela[2].moeda),
+                      trailing: AnimatedOpacity(
+                        opacity: opacidade ? 1 : 0,
+                        duration: Duration(milliseconds: 800),
+                        child: Column(
+                          children: [
+                            Text(tabela[2].valor),
+                            Text(
+                               tabela[2].variacao.toString() + '%',
```

Comment:
é uma boa prática utilizar interpolação nas concatenações, inclusive, interpolando com {} você não vai precisar usar o to.String(), ficaria assim: 
'${tabela[2].variacao}%'

Reply:
Realizada as correções